third_party/terraform/utils/utils.go

```diff
@@ -447,3 +447,11 @@ func paginatedListRequest(baseUrl string, config *Config, flattener func(map[str
 
 	return ls, nil
 }
+
+// For managed SSL certs, if new is an absolute FQDN (trailing '.') but old isn't, treat them as equals.
```

Comment:
Do you mind changing this file to a `.go.erb` and adding guards around this so it's only generated when generating at beta? Travis looks for unused code, and we're not consuming it in tpg yet.

Reply:
Done.